src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -25,8 +27,21 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
      */
     _registerEvents() {
         const checkboxes = DomAccess.querySelectorAll(this.el, this.options.checkboxSelector);
+        const arrowIcons = DomAccess.querySelectorAll(this.el, this.options.arrowIconsSelector);
+        // because some functions need to call inside another object, for example assigning an event.
```

Comment:
A sentence should start with a capital letter.

Reply:
Fixed